components/director/internal/systemfetcher/model.go

```diff
@@ -0,0 +1,58 @@
+package systemfetcher
+
+import (
+	"encoding/json"
+
+	"github.com/tidwall/gjson"
+)
+
+var (
+	// Mappings global static configuration which is set after reading the configuration during startup, should only be used for the unmarshaling of system data
+	Mappings []TempMapping
+)
+
+type TempMapping struct {
+	Name        string
+	SourceKey   []string
+	SourceValue []string
+}
+
+type AdditionalUrls map[string]string
+
+type AdditionalAttributes map[string]string
+type SystemBase struct {
```

Comment:
My OCD spidey sense tells me there should be an empty new line between `AdditionalAttributes` and `SystemBase` 😄